```
ld : error : object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(nethost.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13) [/Users/runner/work/1/s/src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj]
##[error]ld(0,0): error : (NETCORE_ENGINEERING_TELEMETRY=Build) object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(nethost.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13)
ld : error : object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(nethost.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13) [/Users/runner/work/1/s/src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj]
##[error]ld(0,0): error : (NETCORE_ENGINEERING_TELEMETRY=Build) object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(nethost.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13)
ld : error : object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(utils.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13) [/Users/runner/work/1/s/src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj]
##[error]ld(0,0): error : (NETCORE_ENGINEERING_TELEMETRY=Build) object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(utils.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13)
ld : error : object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(utils.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13) [/Users/runner/work/1/s/src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj]
##[error]ld(0,0): error : (NETCORE_ENGINEERING_TELEMETRY=Build) object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(utils.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13)
ld : error : object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(fxr_resolver.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13) [/Users/runner/work/1/s/src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj]
##[error]ld(0,0): error : (NETCORE_ENGINEERING_TELEMETRY=Build) object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(fxr_resolver.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13)
ld : error : object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(fxr_resolver.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13) [/Users/runner/work/1/s/src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj]
##[error]ld(0,0): error : (NETCORE_ENGINEERING_TELEMETRY=Build) object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(fxr_resolver.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13)
ld : error : object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(trace.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13) [/Users/runner/work/1/s/src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj]
##[error]ld(0,0): error : (NETCORE_ENGINEERING_TELEMETRY=Build) object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(trace.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13)
ld : error : object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(trace.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13) [/Users/runner/work/1/s/src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj]
##[error]ld(0,0): error : (NETCORE_ENGINEERING_TELEMETRY=Build) object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(trace.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13)
ld : error : object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(pal.unix.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13) [/Users/runner/work/1/s/src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj]
##[error]ld(0,0): error : (NETCORE_ENGINEERING_TELEMETRY=Build) object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(pal.unix.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13)
ld : error : object file (/Users/runner/work/1/s/.packages/microsoft.netcore.app.host.osx-x64/6.0.3/runtimes/osx-x64/native/libnethost.a(pal.unix.cpp.o)) was built for newer macOS version (10.14) than being linked (10.13) [/Users/runner/work/1/s/src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj]
```

@jkoritzinsky as you added that path to the test project, would you know what needs to be updated to link against >= 10.14? Presumably this path https://github.com/dotnet/runtime/blob/7e408a0fb69487a9113fab16be96fc3ea75a306d/src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj#L75-L76?

--- a/src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj
+++ b/src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/NativeExports.csproj
@@ -73,7 +73,7 @@
     </PropertyGroup>
     <PropertyGroup Condition="'$(TargetOS)' == 'OSX'">
       <TargetTriple Condition="'$(TargetArchitecture)' == 'arm64'">arm64-apple-macos11</TargetTriple>
-      <TargetTriple Condition="'$(TargetArchitecture)' == 'x64'">x86_64-apple-macos10.13</TargetTriple>
+      <TargetTriple Condition="'$(TargetArchitecture)' == 'x64'">x86_64-apple-macos10.14</TargetTriple>
       <XCodeSdkName>macosx</XCodeSdkName>
     </PropertyGroup>
 

